pkg/cmd/util/clientcmd/decorators.go

```diff
@@ -0,0 +1,37 @@
+package clientcmd
+
+import (
+	"fmt"
+	"strings"
+)
+
+func DecorateErrors(err error) error {
```

Comment:
The two methods below don't need to be public.  This isn't really "decoration" - you're just adding wrappers around certain errors.  Just make this "WrapClientErrors" and use a switch statement to select between them.